### ISSUE TITLE
backup triggers keychain popup for password despite env password

### ISSUE THREAD

**Issue by robbat2** [gilbertchen/duplicacy#150]
If a password is provided via environment, Duplicacy should NOT contact the keyring service.

Doing the following causes a keyring password prompt to pop up, but the command continues & completes even while the keyring popup is on the screen.
```
export DUPLICACY_PASSWORD=foobar123
duplicacy init -e  snapshot-id /tmp/storage-dir/
duplicacy backup
```

Doing strace on the backup shows it contacting DBus for the secret service.
```
[pid  3986] connect(4, {sa_family=AF_UNIX, sun_path=@"/tmp/dbus-enuP1HYy8w"}, 23) = 0
[pid  3986] getpeername(4, {sa_family=AF_UNIX, sun_path=@"/tmp/dbus-enuP1HYy8w"}, [112->23]) = 0
[pid  3986] <... read resumed> "REJECTED EXTERNAL DBUS_COOKIE_SHA1 ANONYMOUS\r\n", 4096) = 46
...
[pid  3989] write(4, "l\1\0\1\25\0\0\0\2\0\0\0\220\0\0\0\1\1o\0\30\0\0\0/org/freedesktop/secrets\0\0\0\0\0\0\0\0\6\1s\0\27\0\0\0org.freedesktop.secrets\0\3\1s\0\v\0\0\0OpenSession\0\0\0\0\0\2\1s\0\36\0\0\0org.freedesktop.Secret.Service\0\0\10\1g\0\2sv\0\5\0\0\0plain\0\1s\0\0\0\0\0\0\0\0\0", 181 <unfinished ...>
```

**Comment by gilbertchen** [comment]
Good point.  It really shouldn't.

**Comment by gilbertchen** [comment]
Fixed by https://github.com/gilbertchen/duplicacy/commit/3f83890859ce131ce60f9c81a5a92cb82ade6979

**Comment by tbain98** [comment]
If this issue was fixed via that commit, shouldn't it be transitioned to a Closed state?

**Comment by gilbertchen** [comment]
Usually I want the fix to be comfirmed first before I close it.

**Comment by tbain98** [comment]
Makes sense. I've not developed on GitHub, and I'm used to issue tracking workflows where there's a state that means "implemented but not yet confirmed to work," so I was assuming there was such a state available; sorry for the confusion.

**Comment by robbat2** [comment]
I want to test this, but duplicacy is not building presently, so 
I have to wait for the next release. https://github.com/gilbertchen/duplicacy/issues/56#issuecomment-328365631

**Comment by gilbertchen** [comment]
@robbat2 I don't know what went wrong there but can you try `go get -u ./...` instead?

**Comment by robbat2** [comment]
I got it built now (commit 8808ad5c28), and I'm still being prompted for keychain.

**Comment by gilbertchen** [comment]
Can anyone else confirm whether the fix is working or now?
